### ISSUE TITLE
Bounding Box as a prompt

### ISSUE THREAD

**Issue by Rohinivv96** [IDEA-Research/Grounded-Segment-Anything#236]
Greetings, 

After experimenting with various resources offered in the repository, I am interested in exploring the capabilities of the Grounded-Segment Anything (GSA) model. Specifically, I am curious if it is feasible to input bounding box (BB) coordinates as a prompt instead of a text prompt as in a similar manner to the SAM model. 

I would appreciate your guidance in determining if this feature is available in the GSA model and, if so, the steps necessary to enact it. 

Thank you.

**Comment by rentainhe** [comment]
> Greetings,
> 
> After experimenting with various resources offered in the repository, I am interested in exploring the capabilities of the Grounded-Segment Anything (GSA) model. Specifically, I am curious if it is feasible to input bounding box (BB) coordinates as a prompt instead of a text prompt as in a similar manner to the SAM model.
> 
> I would appreciate your guidance in determining if this feature is available in the GSA model and, if so, the steps necessary to enact it.
> 
> Thank you.

Yes for sure, actually Grounded-SAM is just a simple pipeline by combing GoundingDINO and SAM, for other conditional prompt, you can simply remove GroundingDINO and just use original sam demo for this
